PalasoUIWindowsForms/Progress/LogBox.cs

```diff
@@ -287,7 +287,7 @@ private void Write(Color color, FontStyle style, string msg, params object[] arg
 						rtfBoxForDelegate.SelectionColor = color;
 						rtfBoxForDelegate.SelectionFont = fnt;
 #endif
-						rtfBoxForDelegate.AppendText(string.Format(msg + Environment.NewLine, args));
+						rtfBoxForDelegate.AppendText(msg.FormatWithErrorStringInsteadOfException(args) + Environment.NewLine);
```

Comment:
No unit tests because `SafeInvoke` prevents this line from being executed before the test exists.